tailwind.config.js

```diff
@@ -135,6 +135,12 @@ module.exports = {
       },
       opacity: {
         '10': '0.1'
+      },
+      screens: {
+        'sm-max': { 'max': '639px' },
+        'md-max': { 'max': '767px' },
+        'lg-max': { 'max': '1023px' },
+        'xl-max': { 'max': '1279px' }
```

Comment:
Do we really need these breakpoints? Maybe if my suggestions about mobile-first approach will work fine we could keep things simple and use what we already have?

Reply:
Absolutely not! The mobile-first worked perfectly! I will take it off!